pyblazing/pyblazing/apiv2/context.py

```diff
@@ -912,6 +917,15 @@ def kwargs_validation(kwargs, bc_api_str):
             "max_bytes_chunk_read",  # Used for reading CSV files in chunks
             "local_files",
             "get_metadata",
+            # SQL Engines arguments
+            "from_sql",
+            "hostname",
+            "port",
+            "username",
+            "password",
+            "schema",
+            "table_filter",
```

Comment:
Even when the predicate push down is ready I would vote to leave this "pre-filter" feature there, because it offers more granular ways to create a sql table. cc @drabastomek @roaramburu @felipeblazing